modules/claims_api/lib/library_base.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'claims_api/claim_logger'
+require 'custom_error'
+
+module ClaimsApi
+  class LibraryBase
+    def error_handler(error, async = true) # rubocop:disable Style/OptionalBooleanParameter
+      ClaimsApi::CustomError.new(error, async).build_error
+    end
+
+    def log_outcome_for_claims_api(action, status, response)
+      ClaimsApi::Logger.log(action,
+                            detail: "#{action} #{status}: #{response}")
+    end
+
+    def handle_transaction(service)
+      resp = client.get(service).body[:items]
```

Comment:
Curious if the intent is to use this method for `post` requests as well. If so, would we need a param to indicate the request type + associated logic? If not, should we rename this method to something like `handle_get_request`?